chapter2/nifs.md

```diff
@@ -3,16 +3,22 @@
 NIFs (Native Implemented Functions) are used in Membrane Framework as wrappers for C libraries.
 Since using NIFs has many drawbacks and can even crash Erlang virtual machine during the runtime, we use them only when absolutely necessary or it makes no sense to rewrite a complicated piece of code from scratch.
 
-## Bundlex
+## [Bundlex](https://github.com/membraneframework/bundlex)
 
-To simplify and unify the process of writing and compiling NIFs, we use our own build tool - [Bundlex](https://github.com/radiokit/bundlex). It is a multi-platform tool that provides a convenient way of loading, compiling NIFs in elixir modules. For more information, please visit the Bundlex's github page.
+To simplify and unify the process of writing and compiling NIFs, we use our own build tool - [Bundlex](https://github.com/membraneframework/bundlex). It is a multi-platform tool that provides a convenient way of loading, compiling NIFs in elixir modules. For more information, please visit the Bundlex's github page.
 
+## [Unifex](https://github.com/membraneframework/unifex)
 
-## Membrane Common
+Generally, Writing NIFs require creating a lot of boilerplate code and programmer's efforts. To make this process more pleasant, Membrane Framework exports tool [Unifex](https://github.com/membraneframework/unifex) that is responsible for generating interfaces between simple C libraries and Elixir on the base of short `.exs` files.
+
+A quick introduction to Unifex is available [here](https://github.com/membraneframework/unifex/blob/master/pages/creating-unifex-nif.md).
```

Comment:
/Should be done after publishing unifex/ The link should take the reader to version from `hexdocs.pm`